inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/support/AnnotationQueueOverviewDataProvider.java

```diff
@@ -300,60 +350,111 @@ public long getInProgressAmountForDocument(
 
     //Returns a random document out of all documents in the project.
     //Only a document is chosen which is not yet given to annotators more than the default number
-    //per document number
-    public SourceDocument getRandomDocument()
+    //per document number (TODO),
+    //However, if there was already a document in progress,
+    // this will always be returned (can only be one)
+
+    public SourceDocument getRandomDocument(
```

Comment:
Should be extracted into a strategy and be made configurable via the workflow traits to be added in (one of the next) PRs.